shared/android/app/build.gradle

```diff
@@ -85,8 +85,12 @@ def VERSION_NAME = "5.9.0"
 project.ext.react = [
     // the root of your project, i.e. where "package.json" lives
     root: "../../",
-    enableHermes: KB_USEHERMES == 'true',  // clean and rebuild if changing
-    enableV8: KB_USEV8 == 'true',  // clean and rebuild if changing
+    entryFile: "index.android.js",
+    enableHermes: true, // clean and rebuild if changing
```

Comment:
we'll move forward w/ using hermes and not using v8. hermes is now the default for RN so its just easier to stick with it